framework/include/interfaces/FVRelationshipManagerInterface.h

```diff
@@ -26,7 +26,36 @@ class FVRelationshipManagerInterface
    */
   static void setRMParams(const InputParameters & obj_params,
                           InputParameters & rm_params,
-                          unsigned short ghost_layers);
+                          const unsigned short ghost_layers,
+                          const bool attach_geometric_early);
```

Comment:
AFAICT the only public use of this now is in `FVFluxKernel::adjustRMGhostLayers` which is no longer called by anyone. Can we make this private and delete `FVFluxKernel::adjustRMGhostLayers`?  Also, I believe nobody calls any of these methods with `attach_geometric_early = false` now. Can we delete that parameter from these methods?